### PR TITLE
Date and time

### PR DESCRIPTION
**Pull request by mahdiHashemi14**
Please review and comment / commit if anything is wrong

### CODE REVIEW

File: 1-js/05-data-types/11-date/article.md [hunk 1]
```diff
@@ -144,151 +144,150 @@ The following methods allow to set date/time components:
...
+## بِنچمارک‌گیری (Benchmarking)
 
-If we want a reliable benchmark of CPU-hungry function, we should be careful.
+اگر ما یک بنچمارک قابل اطمینان از تابعی که پردازنده (CPU) را خیلی مشغول می‌کند بخواهیم، باشد مراقب باشیم.
 
-For instance, let's measure two functions that calculate the difference between two dates: which one is faster?
+برای مثال، بیایید دو تابع که تفاوت بین دو تاریخ را محاسبه می‌کنند را اندازه‌گیری کنیم: کدام یک از آنها سریع‌تر است؟
 
-Such performance measurements are often called "benchmarks".
+چنین اندازه‌گیری‌های عملکردی را «بِبنچمارک» می‌گویند.
```
Comment (mahdyar) [inline 1-js/05-data-types/11-date/article.md:270]:
```suggestion
چنین اندازه‌گیری‌های عملکردی را «بِنچمارک» می‌گویند.
```

File: 1-js/05-data-types/11-date/article.md [hunk 2]
```diff
@@ -359,75 +358,75 @@ alert( 'Total time for diffSubtract: ' + time1 );
...
+فراخوانی `Date.parse(str)` رشته را به فرمت داده شده تجزیه می‌کند و مهرزمانی (تعداد میلی‌ثانیه‌های گذشته از اول ژانویه 1970 با UTC+0) را برمی‌گرداند. اگر فرمت قابل قبول نباشد، `NaN` را برمی‌گرداند.
 
-For instance:
+برای مثال:
 
 ```js run
 let ms = Date.parse('2012-01-26T13:51:50.417-07:00');
 
-alert(ms); // 1327611110417  (timestamp)
+alert(ms); // 1327611110417  (مهرزمانی)
```
Comment (mahdyar) [inline 1-js/05-data-types/11-date/article.md:401]: "برچسب زمان" is a better equivalent. [[+](https://fa.wikipedia.org/wiki/%D8%A8%D8%B1%DA%86%D8%B3%D8%A8_%D8%B2%D9%85%D8%A7%D9%86)]

**Review by mahdyar — CHANGES_REQUESTED** [review]

**Comment by mahdiHashemi14** [comment]
Thank you, @mahdyar
/done

**Review by mahdyar — APPROVED** [review]

**Comment by mahdyar** [comment]
Thank you @mahdiHashemi14!

**Pull request merged**